misk/src/main/kotlin/misk/web/actions/WebActionMetadataAction.kt

```diff
@@ -0,0 +1,26 @@
+package misk.web.actions
+
+import misk.security.authz.Unauthenticated
+import misk.web.Get
+import misk.web.RequestContentType
+import misk.web.ResponseContentType
+import misk.web.jetty.WebActionsServlet
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class WebActionMetadataAction : WebAction {
+  @Inject internal lateinit var servlet: WebActionsServlet
+
+  @Get("/api/webactionmetadata")
+  @RequestContentType(MediaTypes.APPLICATION_JSON)
+  @ResponseContentType(MediaTypes.APPLICATION_JSON)
+  @Unauthenticated
```

Comment:
we’ll need to fix these ASAP

Reply:
to add an annotation for admin tab permissions?